### PR TITLE
feat - DCV-1358-include-filter-select-and-exclude-arguments-in-generate-source 

### PR DESCRIPTION
**Pull request by SantiMenendez19**

### CODE REVIEW

File: dbt_coves/tasks/generate/base.py [hunk 1]
```diff
@@ -93,11 +98,27 @@ def get_relations(self, filtered_schemas):
...
                     break
 
+        excluded = []
+        for rel in listed_relations:
+            for selector in rel_excludes:
+                if re.search(selector, rel.name):
+                    excluded.append(rel.name)
+                    break
+        
+        print(excluded)
```
Comment (ssassi) [inline dbt_coves/tasks/generate/base.py:108]: Remove print

File: dbt_coves/tasks/generate/base.py [hunk 2]
```diff
@@ -93,11 +98,27 @@ def get_relations(self, filtered_schemas):
                     rel_name_selectors.append(rel.name)
                     break
 
+        excluded = []
```
Comment (ssassi) [inline dbt_coves/tasks/generate/base.py:101]: It's missing the same transformation done in lines 86 to 89, replacing "*" by ".*"

File: dbt_coves/tasks/generate/base.py [hunk 3]
```diff
@@ -76,8 +76,13 @@ def select_schemas(self, schemas):
...
+            relation for relation in self.get_config_value("select")
         ]
+
+        rel_excludes = [
+            relation for relation in self.get_config_value("exclude")
+        ]
+
         rel_wildcard_selectors = []
         for rel_name in rel_name_selectors:
             if "*" in rel_name:
```
Comment (ssassi) [inline dbt_coves/tasks/generate/base.py:88]: this `if` is not needed

Comment (BAntonellini) [reply dbt_coves/tasks/generate/base.py:88]: Are you sure it's not needed?

```
rel_wildcard_selectors = []
for rel_name in rel_name_selectors:
            if "*" in rel_name:
                rel_wildcard_selectors.append(rel_name.replace("*", ".*"))
```

I think it's managing that only those with wildcard in it's name are added to `rel_wildcard_selectors`

if we remove that `if` all `rel_names` will be added to the list

Comment (ssassi) [reply dbt_coves/tasks/generate/base.py:88]: you're right, disregard the comment

**Pull request merged**